services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -101,6 +100,10 @@ const StateUserRoutes = ({
                     path={RoutesRecord.SUBMISSIONS_NEW}
                     element={<NewStateSubmissionForm />}
                 />
+                <Route
```

Comment:
Please feature flag this new route for state users. There is an edit unlock flag. Thank you!